src/modules/bundler.js

```diff
@@ -16,21 +15,25 @@ async function createAsarFile() {
     const binaryName = configObj.cli.binaryName;
 
     fs.mkdirSync(`temp`, { recursive: true });
-    await fse.copy(`./${resourcesDir}`, `temp/${resourcesDir}`, {overwrite: true});
+    fs.copyFileSync(`./${resourcesDir}`, `temp/${resourcesDir}`)
```

Comment:
According to the [Node.js docs](https://nodejs.org/api/fs.html#fscopyfilesyncsrc-dest-mode) seems like this is not recursively copying directories. Please note that I would like to keep using `fse` for several reasons like easy abstract API, etc